rails/app/views/home/index.html.erb

```diff
@@ -1,14 +1,3 @@
-<div class="row">
-  <% @title = "Coral Gardeners | Home" %>
```

Comment:
It looks like you meant to move the flash behavior to application.html. I don't see this title line over there though.